workers/generic-worker/helper_test.go

```diff
@@ -80,8 +80,8 @@ func scheduleNamedTask[P GenericWorkerPayload | dockerworker.DockerWorkerPayload
 		// horrible hack here, until we have jsonschema2go generating pointer types...
 		//
 		//////////////////////////////////////////////////////////////////////////////////
-		b = bytes.Replace(b, []byte(`"expires":"0001-01-01T00:00:00.000Z",`), []byte{}, -1)
-		b = bytes.Replace(b, []byte(`,"expires":"0001-01-01T00:00:00.000Z"`), []byte{}, -1)
+		b = bytes.ReplaceAll(b, []byte(`"expires":"0001-01-01T00:00:00.000Z",`), []byte{})
+		b = bytes.ReplaceAll(b, []byte(`,"expires":"0001-01-01T00:00:00.000Z"`), []byte{})
```

Comment:
Is this still needed? I thought there was some omitzero struct tag we could use or something... But I forget the details.

Reply:
`omitzero` comes into play with go1.24. Which I still have a WIP PR opened up for https://github.com/taskcluster/taskcluster/pull/7512. Seems to be something to do with the new implementation of `user.Current()` on Windows that's screwing up our tests https://tip.golang.org/doc/go1.24#osuserpkgosuser. Would love to hear your thoughts on this if you ever get a chance to look into that PR and the test failures! 🙏🏻